.github/workflows/generate_pr_gold.yml

```diff
@@ -98,13 +107,6 @@ jobs:
     - name: Update the Gold
       id: update-gold
       run: |
-        source ${GITHUB_WORKSPACE}/.github/actions/common.sh
-        start_group Update Gold Label
-        echo "Old Label: $(ldmx_gold_label)"
-        echo "${GITHUB_REF##refs/tags/}" > ${LDMX_VERSION_FILE}
-        echo "New Label: $(ldmx_gold_label)"
-        git add ${LDMX_GOLD_LABEL_FILE}
-        end_group
```

Comment:
We still need to update the label here because this is in a new job (`pr-new-gold` instead of `generation` or `compile-ldmx-sw`) with a new clone of ldmx-sw and our previous edit of VERSION has not been applied (the `generation` job gets the updated VERSION from the tar-ball of the ldmx-sw build it downloads). I would just keep these lines as is maybe with a comment line about why we need to do both.